src/app/layout.tsx

```diff
@@ -13,8 +13,9 @@ import { authOptions } from "./api/auth/[...nextauth]/route";
 const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "Firefox Monitor",
+  description:
+    "Find out if you’ve been part of a data breach with Firefox Monitor. We’ll help you understand what to do next and continuously monitor for any new breaches.",
```

Comment:
Matches the current values on prod, but will need to confirm w/ a deployed Next.js version to see if we're setting all the `twitter:*` and `og:*` social bits as well. Plus I guess overriding these on the breach detail pages, ie: https://monitor.firefox.com/breach-details/linkedin.

```html
<head>
  <title>Firefox Monitor - LinkedIn Data Breach</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=320, initial-scale=1">
  <meta name="description" content="Use Firefox Monitor to find out if your personal information was exposed in this breach, and understand what to do next.">
  <meta name="twitter:title" content="Were you affected by the LinkedIn Data Breach?">
  …
</head>
```
Although I _think_ we're currently translating these values in the current pre-Next.js site:
https://github.com/mozilla/blurts-server/blob/83a32d11b4a95abbde24088fa9dcdcba2c5fc00a/src/views/guestLayout.js#L12-L15

Reply:
Good points, thanks... yes once it's deployed to dev/stage this will be easy to check. As an aside, we should really start having unique titles per page, but for now I think it's OK. Also yes good point on localization, I'll add that to this PR.

Reply:
Here are the [docs](https://nextjs.org/docs/app/api-reference/functions/generate-metadata), I needed to switch this from using `exprt const metadata ...` to `export async function generateMetadata()` so the `l10n` module can get the headers.

I took a look at what the current Monitor is returning for its meta tags and returned the equivalent above ^ I'll also review our current meta tags against the recommended. Having different titles for different pages is something that was suggested for better SEO, but I'm not as sure about the meta tags, I'll find out.

Reply:
> Good points, thanks... yes once it's deployed to dev/stage this will be easy to check. As an aside, we should really start having unique titles per page, but for now I think it's OK. Also yes good point on localization, I'll add that to this PR.

Oh, I missed that this was fixed in the old app in https://github.com/mozilla/blurts-server/pull/3009 - I will do the same here as well then.

Reply:
Hmm OK, so looks like Google doesn't localise (i.e. it doesn't send `Accept-Language` headers, so it will always get English).

I can't find anything for Twitter cards, but it looks like we'll need to enumerate supported locales and switch locales according to the `fb_locale` query parameter (so much for OpenGraph being independent) to have Facebook actually use the localised versions: https://stackoverflow.com/a/20827962

So given that we don't currently do that on production already, I wouldn't worry too much about localisation.